README.md

```diff
@@ -1,2 +1,253 @@
-# microverse-portfolio
-Microverse portfolio design
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="https://avatars.githubusercontent.com/u/56425107?v=4" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse Portfolio Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+<!--   - [Install](#install) -->
+  - [Usage](#usage)
+<!--   - [Run tests](#run-tests) -->
+<!--   - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+<!-- - [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
```

Comment:
- [OPTIONAL] I think it would be better if  you ensure that all the header/sections of the README template are present in your README

### **_Your README now_** 🔽
![image](https://user-images.githubusercontent.com/106119105/211810127-83424276-1aff-4f6a-9869-b7b0c1bb16f0.png)

### **_Microverse [README template](https://github.com/microverseinc/readme-template)_** 🔽
![image](https://user-images.githubusercontent.com/106119105/211810390-ca3a3348-c349-4085-8b83-2991e1d3eef8.png)